desc/equilibrium/coords.py

```diff
@@ -503,6 +503,132 @@ def fixup(x, *args):
     return out
 
 
+def _L_partial_sum(L, L_lmn):
+    """Convert FourierZernikeBasis to set of Fourier series.
+
+    TODO(#1243) Do proper partial summation once the DESC
+    DESC basis are improved to store the padded tensor product basis.
+    https://github.com/PlasmaControl/DESC/issues/1243#issuecomment-3131182128.
```

Comment:
@PlasmaControl/desc-dev in #1508 header it says "figure out how to do FourierZernike". Basically until the basis is padded https://github.com/PlasmaControl/DESC/issues/1243#issuecomment-3131182128 there is no efficient implementation without loops.